src/probnum/diffeq/odesolution.py

```diff
@@ -89,13 +88,6 @@ def __init__(self, times, rvs, solver):
             self._y = _RandomVariableList(rvs)
         self._solver = solver
 
-    def _proj_normal_rv(self, rv, coord):
-        """Projection of a normal RV, e.g. to map 'states' to 'function values'."""
-        q = self._solver.prior.ordint
-        new_mean = rv.mean[coord :: (q + 1)]
-        new_cov = rv.cov[coord :: (q + 1), coord :: (q + 1)]
-        return Normal(new_mean, new_cov)
-
```

Comment:
For the moment, I removed those functions. I know you wrote them and I am sorry if you don't like it. They are not completely lost, so tell me if you want them back in :) My idea behind removing them was that now the projection is encapsulated in `Integrator`, so in the near future we can make projection functions there and use them here again. Are you okay with that?